src/util/tools.cc

```diff
@@ -447,11 +443,9 @@ void dict_decode_simple(const std::string& url, std::map<std::string, std::strin
 std::string mime_types_to_CSV(std::vector<std::string> mimeTypes)
 {
     std::ostringstream buf;
-    for (size_t i = 0; i < mimeTypes.size(); i++) {
-        if (i > 0)
-            buf << ",";
-        std::string mimeType = mimeTypes[i];
-        buf << "http-get:*:" << mimeType << ":*";
+    for (const auto& mimeType : mimeTypes) {
```

Comment:
Another hint is to modernize for fmt:
https://fmt.dev/latest/api.html#ranges-and-tuple-formatting

something like
`return fmt::format("{}", fmt::join(mimetypes, ","));`

Also consider accepting argument by const &

Reply:
Or maybe not since it will just decorate the first item in the list...